config/longrun_configs/longrun_hs_rhoe_equilmoist_nz63_0M_55km_rs35km.yml

```diff
@@ -6,7 +6,7 @@ z_elem: 63
 dz_bottom: 30.0
 dz_top: 3000.0
 z_max: 55000.0
-kappa_4: 2.0e16
```

Comment:
This run is stable on CPU but unstable on GPU. I decreased the hyperdiffusivity and it became stable. I think we are probably just close to the unstable regime.